### DISCUSSION TITLE
[Error 13] Permission denied.

### DISCUSSION THREAD

**Discussion by saruhan46** [Chia-Network/chia-blockchain#3496] [Support]
I have problem, When I open Chia program ı get this warning " [Error 13] Permission denied. You are trying to access a file/directory without having the necessary permissions. Most likely one of the plot folders in your config.yaml has an issue."
How can I fix ? I have already delete program and  reinstall but not fix.

**Comment by BigUncle** [comment]
Check the `plot_directories` values in  your config.yaml -- which in the directory of `%USERPROFILE%\.chia\mainnet\config`  -- whether you the values(directories) exist. Here is my configuraction:
```
  plot_directories:
  - F:\Plots
  - H:\Plots
  - G:\Plots
  - I:\Plots
  - J:\Plots
  - E:\Plots
```

**Comment by saruhan46** [comment]
My plots has external disc and its has E:/Chai/plots.
Also l delete program and install new version and l add my keyf after its
error fingerprint but l dont use this?

3 May 2021 Pzt 09:45 tarihinde BigUncle ***@***.***> şunu
yazdı:

**Reply by BigUncle** [reply]
`E:/Chai/plots` or `E:/Chia/plots` ?
Please check it again.
It may not be your program problem but your configuration.

**Comment by saruhan46** [comment]
I have problem for private key, I have Install last version of program. but
I write all Import Wallet from Mnemonics Enter the 24 word mnemonic that
you have saved in order to restore your Chia wallet. next time I have
error. attached.
Also I have create new one. I cant open before plots.

BigUncle ***@***.***>, 3 May 2021 Pzt, 17:52 tarihinde şunu
yazdı:

**Reply by thiamkhee** [reply]
you have to provide permission to those drives or folders.
if on windows you can right click -> properties -> security tab

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
